src/main/java/step2/domain/God.java

```diff
@@ -0,0 +1,29 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class God {
+
+    public static List<Integer> issueNumbers(int upperbound, int count) {
+        if (count > upperbound) throw new IllegalArgumentException();
+        List<Integer> generatedNumbers = generateNumbers(upperbound);
+        Collections.shuffle(generatedNumbers);
+
+        List<Integer> chosenNumbers = new ArrayList<>();
+        IntStream.range(0, count)
+                .forEach(val -> chosenNumbers.add(generatedNumbers.get(val)));
+        Collections.sort(chosenNumbers);
+        return chosenNumbers;
+    }
+
+    private static List<Integer> generateNumbers(int upperbound) {
+        List<Integer> generatedNumbers = new ArrayList<>();
+        IntStream.range(1, upperbound + 1)
+                .forEach(generatedNumbers::add);
+        return generatedNumbers;
+    }
```

Comment:
```suggestion
    private static List<Integer> generateNumbers(int upperbound) {
        return IntStream.rangeClosed(1, upperbound)
                .boxed().collect(Collectors.toList());
    }
```

기왕 함수형의 장점을 살려볼거라면, forEach 보다는 map, reduce, filter 와 같은 기능들을 적극 활용해보면 어떨까요?

Reply:
좋은 피드백 감사합니다!